### ISSUE TITLE
Add Contact Section

### ISSUE THREAD

**Issue by k4therin2** [hacsoc/HackCWRU2016-Website#15]

**Comment by wasianish** [comment]
The contact section is in the footer at the bottom

**Comment by k4therin2** [comment]
Shouldn't we have various organizer's contact info like last year?

**Comment by timtadh** [comment]
It is unclear where it is.
![contact](https://cloud.githubusercontent.com/assets/38620/12246817/a3c0c95e-b87e-11e5-8522-9e3c38f67cc3.png)

**Comment by wasianish** [comment]
True, actually thats a good idea, should we have a separate page or just make it below the sponsors.   Should we have just contact or bio's and pictures

**Comment by k4therin2** [comment]
Bios and pix. I'll collect.

**Comment by wasianish** [comment]
I'll start on the html later tonight
